### PR TITLE
[cli] Make the --verbosity option case insensitive

### LINKED ISSUE #3467: Small improvement: make --verbosity <level> case insensitive
**What are you trying to do?**
To debug another issue, I wanted to set the --verbosity level to Debug.

The [Logging levels](https://discourse.ubuntu.com/t/logging-levels/42261) page listed them with an initial capital, which didn't work because multipass wants the level in all small caps.

**What's your proposed solution?**
Of course the mistake was easily found when multipassd failed to start, but wouldn't it be more user friendly if the parameter would ignore casing?
It seems to me a simple .tolower() in the to_logging_level function in cli.cpp::34 should do the trick.

**Additional context**
I'm new to multipass, and not an experienced C++ developer, so if there's good reason not to make this change, feel free to close the issue. I just thought it can potentially save future users a bit of time and frustration.
Also, the '[Configure Multipass’s default logging level](https://discourse.ubuntu.com/t/configure-multipasss-default-logging-level/42260)' page has just been updated to list the levels, and that page does list them in the correct small caps, so that already helps.

### PR DESCRIPTION
**Pull request by nielsreijers**
Fixes #3467

### CODE REVIEW

File: src/daemon/cli.cpp [hunk 1]
```diff
@@ -33,18 +33,21 @@ namespace
...
         return mpl::Level::info;
-    if (value == "debug")
+    if (value_lower == "debug")
         return mpl::Level::debug;
-    if (value == "trace")
+    if (value_lower == "trace")
         return mpl::Level::trace;
 
-    throw std::runtime_error("invalid logging verbosity: " + value.toStdString());
+    throw std::runtime_error("invalid logging verbosity: " + value.toStdString() +
```
[the remark this answers is not in the document]
Comment (nielsreijers) [reply src/daemon/cli.cpp]: Thanks for the review @sharder996!
You're right, that is more consistent with what I see elsewhere in the code so I've changed it to use fmt::format here as well.

[the remark this answers is not in the document]
Comment (sharder996) [reply src/daemon/cli.cpp]: Thanks!

**Comment by sharder996** [comment]
@nielsreijers Have you had a chance to sign the Canonical cla?

https://ubuntu.com/legal/contributors

**Comment by nielsreijers** [comment]
> @nielsreijers Have you had a chance to sign the Canonical cla?
> 
> https://ubuntu.com/legal/contributors

Ah, I didn't think that was required since my last PR got merged without it.
I'm happy to sign it of course and was just about to do so, but it's asking me for a "Canonical Project Manager or contact"... Any idea what I should fill in there?

**Comment by townsend2010** [comment]
Hi @nielsreijers,

I'm going to jump in here about the CLA :slightly_smiling_face: 

So, the CLA allows for not having to be signed for very small contributions, such as yours with the one line change to the `control` file.  However, if your proposed change is more than a line or two, then the CLA will need to be signed.  You can put me down as the contact: Chris Townsend <christopher.townsend@canonical.com>

Once you have signed it, just let us know here that you did and we can proceed.  Thank you for your contributions!

**Comment by nielsreijers** [comment]
> Hi @nielsreijers,
> 
> I'm going to jump in here about the CLA 🙂
> 
> So, the CLA allows for not having to be signed for very small contributions, such as yours with the one line change to the `control` file. However, if your proposed change is more than a line or two, then the CLA will need to be signed. You can put me down as the contact: Chris Townsend [christopher.townsend@canonical.com](mailto:christopher.townsend@canonical.com)
> 
> Once you have signed it, just let us know here that you did and we can proceed. Thank you for your contributions!

Thanks Chris, I just singed the CLA :-)

**Comment by nielsreijers** [comment]
> > Hi @nielsreijers,
> > I'm going to jump in here about the CLA 🙂
> > So, the CLA allows for not having to be signed for very small contributions, such as yours with the one line change to the `control` file. However, if your proposed change is more than a line or two, then the CLA will need to be signed. You can put me down as the contact: Chris Townsend [christopher.townsend@canonical.com](mailto:christopher.townsend@canonical.com)
> > Once you have signed it, just let us know here that you did and we can proceed. Thank you for your contributions!
> 
> Thanks Chris, I just singed the CLA :-)

Hmm, not sure why it's still failing... I did see the "Thank you" message saying I signed the CLA...
Is it because the first time I filled in my email as the launchpad id? I just filled it in again with nielsreijers as my username, maybe that helps?

**Review by sharder996 — APPROVED** [review]
LGTM!

Thanks for the contribution!

**Comment by sharder996** [comment]
> > > Hi @nielsreijers,
> > > I'm going to jump in here about the CLA 🙂
> > > So, the CLA allows for not having to be signed for very small contributions, such as yours with the one line change to the `control` file. However, if your proposed change is more than a line or two, then the CLA will need to be signed. You can put me down as the contact: Chris Townsend [christopher.townsend@canonical.com](mailto:christopher.townsend@canonical.com)
> > > Once you have signed it, just let us know here that you did and we can proceed. Thank you for your contributions!
> > 
> > 
> > Thanks Chris, I just singed the CLA :-)
> 
> Hmm, not sure why it's still failing... I did see the "Thank you" message saying I signed the CLA... Is it because the first time I filled in my email as the launchpad id? I just filled it in again with nielsreijers as my username, maybe that helps?

The Github action for checking the CLA is broken at the moment, but your confirmation that you have signed it is good enough right now.

**Pull request merged**